output/cloud/expv2/output.go

```diff
@@ -73,6 +73,17 @@ func (o *Output) SetTestRunStopCallback(stopFunc func(error)) {
 func (o *Output) Start() error {
 	o.logger.Debug("Starting...")
 
+	if o.config.MetricPushConcurrency.Int64 <= 1 ||
+		o.config.MetricPushConcurrency.Int64 > 10 {
+		// TODO: the remote service sets this option with a very high value
+		// using the config overwriting feature. It is required for v1 so for reusing the
+		// same option here, it is necessary to force a smaller default value
+		// or use another the provided  if it is in acceptable range.
+		//
+		// Drop the following hack when the migration from v1 is completed.
+		o.config.MetricPushConcurrency = null.IntFrom(4)
+	}
+
```

Comment:
I am not certain I follow this. 

AFAIK there are no problems with this being 1 or with this being very large. 

goroutines are cheap and even with 1 concurrency this will likely work pretty okay.

So I prefer that we only do this if there is actually a reason to do this.

The backend can change what it returns ... probably based on the version (in the headers) in the original request if we want to have different values.

Reply:
I would like to test with values close to what should be the potential final value. And having `20` as final numbers sounds a magic number to me.

>The backend can change what it returns 

I would avoid it, the cost is not balanced, so let's use 20 for now.